src/js/video.js

```diff
@@ -569,5 +569,9 @@ videojs.url = Url;
 
 videojs.defineLazyProperty = defineLazyProperty;
 
+// Adding less ambiguous text for fullscreen button.
+// In a major update this could become the default text and key.
```

Comment:
added to our major version tracking list https://github.com/videojs/video.js/issues/7068